src/components/AgentTree/index.tsx

```diff
@@ -289,7 +292,7 @@ class AgentTree extends React.Component<AgentTreeProps> {
                                     </Text>
                                 </>
                             }
-                            expandByDefault={!nodeData.color}
+                            expandByDefault={childrenHaveDifferentColors}
```

Comment:
I do not know but I can ask

Reply:
from Blair: "afaik the data for that one only has 3 agent types: "Antigen", "Antibody#Unbound", and "Antibody#Bound", so I don't know where the color got set for "Antibody", I would guess somewhere on the front end?"

in the trajectory file:
```
"typeMapping": {"0": {"name": "Antigen", "geometry": {"displayType": "OBJ", "url": "https://aics-simularium-data.s3.us-east-2.amazonaws.com/meshes/obj/binding-affinity/Antigen.obj", "color": "#fd4cc1"}}, 
"1": {"name": "Antibody#Unbound", "geometry": {"displayType": "OBJ", "url": "https://aics-simularium-data.s3.us-east-2.amazonaws.com/meshes/obj/binding-affinity/Antibody.obj", "color": "#4cfdfd"}}, 
"2": {"name": "Antibody#Bound", "geometry": {"displayType": "OBJ", "url": "https://aics-simularium-data.s3.us-east-2.amazonaws.com/meshes/obj/binding-affinity/Antibody.obj", "color": "#ffdf00"}}}, 
```

Reply:
yeah I'm saying this seems like a bug since we already had this check that is supposed to be opening up the children when the colors are different. Instead of changing this code, I would look up further to see what's going wrong